### ISSUE TITLE
Unable to create new repositories using the menu with a screen reader

### ISSUE THREAD

**Issue by stormdragon2976** [gogits/gogs#4292]
The menu to create a new repository, and possibly other stuff, (I'm not sure what all is in it), is not accessible. I use the screen reader Orca, and the options in the menu do not show up. I am not able to find them using lists of links or clickables. It would be greate if these were standard html elements like links.

**Comment by Unknwon** [comment]
Hi, by "The menu to create a new repository", which mean you're referring to?

**Comment by stormdragon2976** [comment]
It says create new, and I think it is supposed to have repositry, organization, and maybe some other stuff. It's on the dash board once you have logged in.

**Comment by stormdragon2976** [comment]
yes, those things are supposed to show up. Using a screen reader, my options are to hit enter to activate it. When I do that, the bits that pop up are not accessible. I'm not even sure they actually pop up.

**Comment by Unknwon** [comment]
Hmm... I'm sorry but I have very hard time understanding what should be displayed as what in terms of screen reader, hope someone with expertise could help. All accessibility issue discussion should go https://github.com/gogits/gogs/issues/3340.
